File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] res = {-1,-1};
        for(int i=0; i<nums.length; i++)
        {
            for(int j=i+1; j<nums.length; j++)
            {
                if(nums[i]+nums[j]==target)
                {
                    res[0]=i;
                    res[1]=j;
                }
            }
        }
        return res;
    }
}

File: Data Structures & Algorithms/subsets/submission-0.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        res = []
        def sub(nums, temp, i):
            if i == len(nums):
                res.append(temp[:])
                return
            sub(nums,temp,i+1)
            sub(nums, temp + [nums[i]] , i+1)

        sub(nums, [], 0)
        print(res)
        return res